### PR TITLE
Fix stats for stork app backup

### PR DESCRIPTION
**Pull request by gejain**
Fix stats for stork app backup

**What this PR does / why we need it**:
Stork event should name based on stork event name, just changed the same for stork app backup

### CODE REVIEW

File: tests/testTriggers.go [hunk 1]
```diff
@@ -684,7 +684,7 @@ func updateLongevityStats(name, eventStatName string, dashStats map[string]strin
 	name = strings.Split(name, "<br>")[0] //discarding the extra strings attached to name if any
 	version, err := Inst().V.GetDriverVersion()
 	product := "px-enterprise"
-	if eventStatName == stats.AsyncDREventName || name == stats.MetroDREventName || name == stats.StorkApplicationBackupEventName {
+	if eventStatName == stats.AsyncDREventName || eventStatName == stats.MetroDREventName || eventStatName == stats.StorkApplicationBackupEventName {
```
Comment (kphalgun-px) [inline tests/testTriggers.go:687]: https://github.com/portworx/torpedo/pull/2137#discussion_r1510020763

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/testTriggers.go (hunk 1 above)
Comment (gejain) [reply tests/testTriggers.go:687]: Yeah Thanks, Its not pushing correctly with name.

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**